src/main/java/com/diffblue/cover/annotations/generated/MethodUnderTest.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Copyright 2024 Diffblue Limited.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ */
+package com.diffblue.cover.annotations.generated;
```

Comment:
Certainly makes sense to separate this from others which provide "testing advice". I can't quite decide whether I like `.generated` but I don't have an obvious improvement to offer.

I would ask where we think `@MaintainedByDiffblue` or whatever will live and consider matching it, but I think we concluded not to bother with that.

Reply:
Thank you, I'm also undecided about the name but can't think of a better one. Perhaps @peterschrammel might help?

Reply:
I moved it to the main package with the other ones, Javadoc explains the difference enough (pointed out by Johannes and Rob) and the name "generated" is I guess more questionable than whether it's a new package or not. Peter was also more on the side of keeping it in the same package. End of bike shed 😄